CryptomatorCommon/Sources/CryptomatorCommonCore/Manager/VaultDBManager.swift

```diff
@@ -382,7 +411,7 @@ public class VaultDBManager: VaultManager {
 	}
 
 	func postProcessChangePassphrase(masterkeyFileData: Data, forVaultUID vaultUID: String, vaultConfigToken: Data?, newPassphrase: String) throws {
-		let cachedVault = CachedVault(vaultUID: vaultUID, masterkeyFileData: masterkeyFileData, vaultConfigToken: vaultConfigToken, lastUpToDateCheck: Date())
+		let cachedVault = CachedVault(vaultUID: vaultUID, masterkeyFileData: masterkeyFileData, vaultConfigToken: vaultConfigToken, lastUpToDateCheck: Date(), masterkeyFileLastModifiedDate: nil, vaultConfigLastModifiedDate: nil)
```

Comment:
Should these dates really be set to `nil`? Wouldn't this cause the cache to be invalidated with the next unlock?